features/README.md

```diff
@@ -41,9 +41,8 @@ file.
 ## Issues
 
 The documentation for rspec-rails is a work in progress. We'll be adding
-Cucumber features over time, and clarifying existing ones.  If you have
-specific features you'd like to see added, find the existing documentation
-incomplete or confusing, or, better yet, wish to write a missing Cucumber
-feature yourself, please [submit an
-issue](http://github.com/rspec/rspec-rails/issues) or a [pull
-request](http://github.com/rspec/rspec-rails).
+Cucumber features over time, and clarifying existing ones.  If you have specific
+features you'd like to see added, find the existing documentation incomplete or
+confusing, or, better yet, wish to write a missing Cucumber feature yourself,
+please [submit an issue](https://github.com/rspec/rspec-rails/issues) or a [pull
+request](https://github.com/rspec/rspec-rails).
```

Comment:
Reformatted to 80-char width

Reply:
Can you undo this, its formatted for relish.